### PR TITLE
First iteration of FIM documentation

### PR DESCRIPTION
**Pull request by arirubinstein**
First iteration of FIM doc page, now with updated config schema.  for issue #1014

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

**Comment by wxsBSD** [comment]
While only tangentially related the YARA work I've been doing may be able to fit in with this, if you are willing to expand it a bit. What are your thoughts on it? I've put together some notes on it that I intend to turn into documentation similar to this and would love either an opinion on it or, if it makes sense, including it here.

https://gist.github.com/wxsBSD/a3ba7f4733125813e58a

**Comment by arirubinstein** [comment]
I updated this with a fix for a bunch of broken links in the docs.  Tested locally, but it should work on rtd

**Comment by marpaia** [comment]
@wxsBSD I like this model of documentation collaboration. Can you throw up the yara docs into a PR and lets iterate on them. The inline comments are really useful.

File: docs/wiki/deployment/file-integrity-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    },
+    "file_changes": {
+      "query": "select * from file_changes;",
+      "interval": 300
+    }
+  },
+  "file_paths": {
+    "homes": [
+      "/root/%%",
+      "/home/apprunner/%%"
```
Comment (marpaia) [inline docs/wiki/deployment/file-integrity-monitoring.md]: maybe `/home/%/%%` to show that you can monitor homedirs more generically?

File: docs/wiki/deployment/file-integrity-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    ],
+    "all_etc": [
```
Comment (marpaia) [inline docs/wiki/deployment/file-integrity-monitoring.md]: Maybe change this to just `etc` to be consistent with `/tmp/%%` being just `tmp`?

File: docs/wiki/deployment/file-integrity-monitoring.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    "tmp": [
+      "/tmp/%%"
+    ]
+  }
+}
+```
+
+### Sample output
+
+As file changes happen, events will appear in the `file_changes` table.  During
```
Comment (marpaia) [inline docs/wiki/deployment/file-integrity-monitoring.md]: Relevant: https://github.com/facebook/osquery/issues/1049

we're going to change this to `file_events` before we release this.

**Comment by marpaia** [comment]
@arirubinstein this is awesomeeeeeeee

**Comment by arirubinstein** [comment]
All updated based on feedback @marpaia

**Comment by marpaia** [comment]
LGTM. I'm down to merge now, or we can wait until we actually change the name of `file_changes` to `file_events`.

**Comment by theopolis** [comment]
This is wonderful!

Let's let this bake for 24hours while we profile/regress-test/apply fixups tomorrow in preparation of a 1.4.5 tag. We can move `file_changes` to `file_events` easily tomorrow morning too.

**Comment by theopolis** [comment]
I commendeered: https://github.com/facebook/osquery/pull/1072#merged-event

**Pull request closed without merging**